jis-api/src/main/java/pl/edu/uj/JImageStream/api/ImageStream.java

```diff
@@ -104,5 +105,29 @@ public ImageStream setThreads(int numberOfThreads) {
         return collector.collect(imageCopy);
     }
 
+    public ImageStream resize(int x, int y){
```

Comment:
It can't work like that, calling stream.apply(filter1).resize(200, 200).apply(filter2).collect() has to execute in a proper order, instead of first resizing image. 

Resizing class should implement ImageTransform. Also, we need to fix height and width in AbstractImageTransform so it sets on start of a filter instead of on creation